### DISCUSSION TITLE
YOLO, but the pull request was merged with a review

### DISCUSSION THREAD

**Discussion by l2dy** [community/community#19885] [Other Feature Feedback, Questions, & Ideas]
My profile has the YOLO badge, but the referenced pull request was merged *with* a review. Is it just about merging a pull request I've submitted myself, regardless of reviews?

**Comment by thomscoder** [comment]
Hi @l2dy 
I've done some investigation and looks like a bug to me since the merge happened after the approval.
The only thing that I can think of is [Protected Branches](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-pull-request-reviews-before-merging). I don't know if you guys at Macports have mandatory reviews before merge. If yes, then I think it is a bug

**Comment by mtak** [comment]
Same problem here, did a PR on a private repo, the PR was approved before merge, yet it still fired:
![image](https://user-images.githubusercontent.com/1315327/177504361-f8d6c7c2-4701-4a91-9fa2-65bd844d4f02.png)

The branch I was merging into has protections (and no, I'm not an admin on the repo):
![branch_protection](https://user-images.githubusercontent.com/1315327/177504603-5a1e2350-70d0-4e9c-98bc-f3830619df31.png)

**Reply by thomscoder** [reply]
> Same problem here, did a PR on a private repo, the PR was approved before merge, yet it still fired: ![image](https://user-images.githubusercontent.com/1315327/177504361-f8d6c7c2-4701-4a91-9fa2-65bd844d4f02.png)
> 
> The branch I was merging into has protections (and no, I'm not an admin on the repo)

Interesting, are you part of an organization?

**Reply by mtak** [reply]
I am, and this was in a private organization repo.

**Comment by azrdn** [comment]
That happened to me as well

**Comment by oliviagunton** [comment]
I wonder if it's happening in cases where there's an automatic (non-blocking) review request to a "codeowners" group and then someone outside that group ends up being the approver? That was true for my incorrectly-flagged PR merge.

**Comment by radar-blip** [comment]
I got YOLO too. Didn't ask for it, didn't want it, and certainly didn't try to achieve it. 
It was all an accident. Wish I could give it back.
